api/v1alpha1/fenceagentsremediation_types.go

```diff
@@ -47,9 +59,9 @@ type FenceAgentsRemediation struct {
 	Status FenceAgentsRemediationStatus `json:"status,omitempty"`
 }
 
-//+kubebuilder:object:root=true
-
+// +kubebuilder:object:root=true
 // FenceAgentsRemediationList contains a list of FenceAgentsRemediation
+// +operator-sdk:csv:customresourcedefinitions:resources={{"FenceAgentsRemediation","v1alpha1","fenceagentsremediations"}}
```

Comment:
IDK why but I assumed that this line would add `FenceAgentsRemediation` to the CSV as `FenceAgentsRemediationTemplate` has been added by [similar line](https://github.com/medik8s/fence-agents-remediation/pull/16/files#diff-9f945e8a1bb4d6890949b218fb8e67f95bb5f04b75cbea808da1291121af2dbaR49) to the [CSV](https://github.com/medik8s/fence-agents-remediation/pull/16/files#diff-370992a193637c0aec12471688d0a6310eb7ce759bcd5aaa18d4e0d522108afdR43-R46).

Reply:
IIUC it's not the case because `FenceAgentsRemediation` is already part of the [CSV](https://github.com/medik8s/fence-agents-remediation/pull/16/files#diff-370992a193637c0aec12471688d0a6310eb7ce759bcd5aaa18d4e0d522108afdR22-R24)

Reply:
`FenceAgentsRemediation` is indeed part of the CSV.
My doubt was about the `resources` field which is missing for `FenceAgentsRemediation` in comparison to `FenceAgentsRemediationTemplate` in [these lines](https://github.com/medik8s/fence-agents-remediation/blob/main/bundle/manifests/fence-agents-remediation.clusterserviceversion.yaml#L59-L103).
I have compared it with [SNR CSV](https://github.com/medik8s/self-node-remediation/blob/main/bundle/manifests/self-node-remediation.clusterserviceversion.yaml#L52-L99), and I see `resources` field per CRD, unlike for FAR...

Reply:
I think that tag needs to go to the FenceAgentsRemediation struct, not the list

Reply:
Yes, it made the trick 😄 
thanks @slintes